src/lib.rs

```diff
@@ -62,6 +63,17 @@ impl Config {
             assets,
         })
     }
+
+    /// Returns the directory where output files should be placed
+    pub fn output_path(&self) -> PathBuf {
+        self.manifest_path.join("target").join("doc")
+    }
+
+    /// Open the generated docs in a web browser.
+    pub fn open_docs(&self) -> Result<()> {
+        open::that(self.output_path().join("index.html")).unwrap(); // TODO: Error handling
```

Comment:
It doesn't return an io::Result, and I wasn't sure how to map its error type to an io::Error.